about.html

```diff
@@ -0,0 +1,118 @@
+  <link rel="stylesheet" href="./styles/general.css">
+  <link rel="stylesheet" href="./styles/headline.css">
+  <link rel="stylesheet" href="./styles/program.css">
+  <link rel="stylesheet" href="./styles/musicians.css">
+  <link rel="stylesheet" href="./styles/footer.css">
+  <link rel="stylesheet" href="./styles/logoInfo.css">
+  <link rel="stylesheet" href="./styles/pastConecerts.css">
+</head>
+<body class="greyBackground">
+  <nav class="navBar">
+    <button class="hambugerMenu"  id="hamburgerMenu">
+      <img class="hambugerMenuImg" src="./assets/navbar/Icon - Menu.png" alt="Mobile menu">
+    </button>
+    <ul class="navbarList">
+      <li class="navbarListItem navbarFirst">
+        <a href="./index.html"><img class="orchestraIcon" src="./assets/navbar/orchestraIcon.jpg" alt="Cordoba Spring Festival 2023 Icon"></a>
+      </li>
+      <li class="navbarListItem"><a class="navbarLink" href="./about.html">About</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Program</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Join</a></li>
+      <li class="navbarListItem"><a class="navbarLink">Sponsor</a></li>
+      <li class="navbarListItem"><a class="navbarLink">News</a></li>
+      <li class="navbarListItem lastItem"><a class="navbarLink">Spring Festival</a></li>
+    </ul>
+  </nav>
+  <main>
+    <section class="headlineAbout">
+      <div class="headlineDivAbout">
+        <p class="smallTitle">"Hello! Music world"</p>
+        <h1 class="pageTitleAbout">CORDOBA CLASICAL<br/> SPRING FESTIVAL <br class="breakMobile"/>2023</h1>
+        <p class="aboutDescription">The Global Summit brings together the community of experts, academics, and activist who comprise
+          the Creative Commons affiliate networks in a different country every two.
+          This year, we hope to expand our invitations list including organizations
+          and individuals
+        </p>
+        <p class="aboutDescriptionDesktop">
+          Experience the magic of classical music at the Cordoba Spring Festival, as we celebrate
+           the brilliance of Beethoven, Mozart, and Handel. From powerful symphonies to exquisite concertos, 
+           immerse yourself in the enchanting world of these iconic composers. Join us for a series of captivating 
+           performances by talented musicians in the vibrant ambiance of Cordoba, 
+          creating unforgettable musical moments that resonate with the spirit of spring.
+        </p>
+        <p class="conctactInfo">Please contact us per Email for any further questions <br class="breakMobile"/>
+        Coroba Clasical Spring Festival</p>
+        <a class="email">cordobaclasicalfestival@gmail.cordoba.gob</a>
+      </div>
+    </section>
+    <section class="logoInfo" id="sectionLink1">
+      <div class="logoDescription">
+        <h2 class="logoDescriptionTitle">Cordoba<br class="breakMobile"/> Clasical Spring Festival Logo</h2>
```

Comment:
- [ ] Kindly add the red line under the title as in the provided design(see image)
![Screenshot from 2023-06-29 02-32-43](https://github.com/IsmaelMastronardi/Capstone1/assets/91142432/1477259f-0147-4246-8d2c-910c01a7c7e6)